### PR TITLE
Adding a default time to environment vars

### PR DESCRIPTION
**Pull request by joshdvir**

### CODE REVIEW

File: app/controllers/health_monitor/health_controller.rb [hunk 1]
```diff
@@ -10,10 +10,9 @@ class HealthController < ActionController::Base
     def check
       res = HealthMonitor.check(request: request)
 
-      unless HealthMonitor.configuration.environmet_variables.nil?
```
Comment (lbeder) [inline app/controllers/health_monitor/health_controller.rb]: `HealthMonitor.configuration.environmet_variables` can be `nil?` indeed?, so perhaps the check should be restored or rewritten as `||= {}`?

In order to accommodate to the existing style, could you change:
- `merge({ 'time' => Time.now.to_s(:db) })` 
to:
- `merge(time: Time.now.to_s(:db))`
?

**Comment by lbeder** [comment]
Thanks for the contribution! 👍

File: app/controllers/health_monitor/health_controller.rb (hunk 1 above)
Comment (joshdvir) [inline app/controllers/health_monitor/health_controller.rb]: HealthMonitor.configuration.environmet_variables can be nil, sure, but it has no meaning even if it is nil we are configuring a default value.

I've changed the styling.

Comment (lbeder) [inline app/controllers/health_monitor/health_controller.rb]: I mean, calling `merge` from `nil` will result in `NoMethodError: undefined method merge for nil:NilClass`, no?

Comment (lbeder) [inline app/controllers/health_monitor/health_controller.rb]: Yeah... I can see that there is a test for that and it's failing now... :(

Comment (joshdvir) [inline app/controllers/health_monitor/health_controller.rb]: yes you are right, fixing it now :)

Comment (lbeder) [inline app/controllers/health_monitor/health_controller.rb]: 👍

**Comment by lbeder** [comment]
Could you just fix the failing tests (just the introduction of the default `time` key) and I'll merge?

**Comment by joshdvir** [comment]
tests are fixed now :)

**Comment by joshdvir** [comment]
Also fixed al the styling issues Robocop loves

**Pull request merged**